samples/bluetooth/direct_test_mode/src/dtm_hw_config.h

```diff
@@ -11,8 +11,8 @@
 extern "C" {
 #endif
 
-#if defined(NRF5340_XXAA_NETWORK) || defined(NRF52833_XXAA) || \
-	defined(NRF52811_XXAA) || defined(NRF52820_XXAA)
+#if defined(CONFIG_HAS_HW_NRF_RADIO_BLE_DF) && \
```

Comment:
This config was removed as part of PR that added radio node to all nrfXXX.dtsi files in Zephyr.
You can use `dfe-supported` property of `radio` node.

Reply:
When is this expected sdk-zephyr?

Reply:
@ppryga : Is this functionality expected for sdk-zephyr at all? If not, I see no reason for blocking this PR. At this time, you will get compilation errors on many nRF52-devices in a sample _required_ for bluetooth certification.